tests/ci-helper.test.ts

```diff
@@ -35,38 +35,39 @@ async function getSMTPInfo():
 
 class TestCIHelper extends CIHelper {
     public ghGlue: GitHubGlue;      // not readonly reference
-    public addPRComment: any;
-    public updatePR: any;
+    public addPRComment = jest.fn();
+    public updatePR = jest.fn();
 
     public constructor(workDir?: string, debug = false, gggDir = ".") {
         super(workDir, debug, gggDir);
         this.testing = true;
         this.ghGlue = this.github;
-        this.addPRComment = jest.fn();
-        this.ghGlue.addPRComment = this.addPRComment;
-        this.updatePR = jest.fn();
-        this.ghGlue.updatePR = this.updatePR;
+        Object.assign(this.ghGlue, this);
         // need keys to authenticate
         // this.ghGlue.ensureAuthenticated = async (): Promise<void> => {};
     }
 
-    public setGHgetPRInfo(o: IPullRequestInfo): void {
+    public setGHGetPRInfo(o: IPullRequestInfo): void {
```

Comment:
The case was matching the method being overridden.  Is that not preferred?

Reply:
You mean since the method name to be overridden is called `getPRInfo()`, you wanted to just prefix `setGH` before it? I see where you're coming from, but I think we should adhere to the camelCasing strategy which upcases _every_ first letter of strings that are appended.

It's like constructing a method name that appends a commit note. Both "commit" and "note" are all-lowercase, yet you would immediately up-case the "c" and the "n" to arrive at the name `appendCommitNote()`. Likewise, I want to up-case the "g" in `getPRInfo()` to arrive at the name `setGHGetPRInfo()`.